monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md

```diff
@@ -0,0 +1,25 @@
+# Uma carteira segura é uma carteira feliz
+
+Sua carteira Monero (XMR) é como um cofre que guarda seus XMR e os mantém seguros. Se você quer ter certeza que isso se mantenha dessa forma, existem algumas boas práticas que você poderá seguir para manter sua carteria o mais segura possível.
+
+Carteiras "quentes" são carteiras que estão conectadas na internet, enquanto carteiras "frias" não. Geralmente as carteiras frias são mais seguras, mas ela sacrificam a conveniência.
+
+Uma carteira "hard", como as populares Ledger e Trezor podem ser vistas como uma mistura entre carteiras quentes e frias pois você pode conecta-la em um dispositivo com internet para enviar ou receber XMR, mas uma vez desconectadas elas não serão acessíveis via internet.
+
+### Aqui vão algumas boas práticas gerais para se ter conhecimento quanto a segurança das carteiras:
+
+1. Para evitar golpes que eventualmente roubem seus XMR, somente use carteiras aprovadas pela comunidade Monero. Um bom ponto de partida é a seção de informações do r/monero que pode ser acessada aqui: https://www.reddit.com/r/Monero/.
+
+2. Se estiver configurando uma carteira fria (por exemplo uma carteira "em papel") para guardar a longo prazo, faça sua pesquisa e siga um dos tutoriais online que têm os comentários e as avaliações mais positivas. Como nestes links do Reddit:
+
+  a. Tutorial de como enviar para uma carteira em papel (Sending to Paper Wallet Tutorial)
+r/Monero/comments/7t5tyx/tutorial_sending_xmr_to_a_paper_wallet_generated
+
+  b. Guia para gerar uma carteira fria em papel em segurança (Guide for Securely Generating an Cold Paper Wallet)
+r/Monero/comments/48cgmd/an_extensive_guide_for_securely_generating_an
+
+3. Considere manter um carteira fria para os XMR que você pretende guardar por um tempo e uma carteira fria para os XMR que você pretende gastar ou negociar. Dica: Carteira fria=conta poupança, Carteira quente=conta corrente.
+
+4. Visite fóruns online e pergunte para aprender as diferentes maneiras que as pessoas mantém suas chaves privadas ou sementes mnemônicas protegidas e seguras, depois escolha a opção que melhor funcione para você. Comece aqui: https://getmonero.org/community/hangouts/. Tem alguma estratégia é melhor do que não ter nenhuma. Já percebeu que pesquisar é um tema importante?
```

Comment:
This link can also be changed:

https://www.getmonero.org/pt-br/community/hangouts/